mlton/xml/xml-tree.fun

```diff
@@ -210,11 +210,11 @@ in
       seq [Con.layout con,
            case arg of
               NONE => empty
-            | SOME t => seq [str " of ", Type.layout t]]
+            | SOME t => seq [str " of ", Type.layoutFormal t]]
    fun layoutTyvars ts =
       case Vector.length ts of
          0 => empty
-       | 1 => seq [Tyvar.layout (Vector.first ts), str " "]
+       | 1 => seq [str "(", Tyvar.layout (Vector.first ts), str ") "]
```

Comment:
Can eliminate the special case for 1-tuple, since the default will have the same behavior.